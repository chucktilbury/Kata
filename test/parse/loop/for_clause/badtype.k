;-------------------------------------------------------------------------------
; Demonstrate bad type name error
;-------------------------------------------------------------------------------

func name ()() {
    for ( 123 some_name in something != 0 ) {

    }
}
