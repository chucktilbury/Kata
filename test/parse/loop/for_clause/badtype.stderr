Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badtype.k: 6: 14: expected a ')', a type name, or a SYMBOL but got literal signed
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badtype.k: 6: 14: expected function body element but got for
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badtype.k: 6: 14: expected function body but got for
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badtype.k: 6: 14: expected module item or end of file but got func
Syntax: /home/chuck/Src/Kata/test/parse/loop/for_clause/badtype.k: 6: 14: module cannot be empty
